viam-cartographer/src/slam_service/slam_service.cc

```diff
@@ -74,24 +74,26 @@ ::grpc::Status SLAMServiceImpl::GetPosition(ServerContext *context,
 
     // Rotate pose to XZ plane. Additional angle offset is used so rotations
```

Comment:
1. Can we please remove these comments and grep the codebase for any other mention of the XZ plane & remove them?
2. Can we please remove `pcdRotation` if it is no longer used?
3.  Can we please remove `pcdOffsetRotation` if it is no longer used?

Reply:
Removed all comments and all association with XZ/rotations